tests/src/lib.rs

```diff
@@ -202,7 +201,6 @@ async fn ws_more_request_than_buffer_should_not_deadlock() {
 		}));
 	}
 
-	concurrent_tx.send(()).unwrap();
```

Comment:
The comment on line 196 isn't correct anymore; we can use any number of requests I think. :)